### ISSUE TITLE
Acroform Field not rotating with document

### ISSUE THREAD

**Issue by pgodwin** [mozilla/pdf.js#9596]
Acrobat form fields are not rotated correctly on the page.

Attach (recommended) or Link to PDF file here:
[private-patient-hospital-claim.pdf](https://github.com/mozilla/pdf.js/files/1836330/private-patient-hospital-claim.pdf)

Configuration:
- Web browser and its version: 65.0.3325.181 (Official Build) (64-bit)
- Operating system and its version: Windows 2016
- PDF.js version: pdfjs-1.10.88-dist
- Is a browser extension: No

Steps to reproduce the problem:
1. Set `renderInteractiveForms` to `true` on the viewer
2. Open the attached PDF. The two Acrobat fields have the wrong rotation applied.

What is the expected behavior? (add screenshot)
Adobe Reader 11:
![image](https://user-images.githubusercontent.com/1046558/37754539-a38b699e-2ded-11e8-9277-a85a85364134.png)

What went wrong? (add screenshot)
Text fields did not rotate with the page correctly:
![image](https://user-images.githubusercontent.com/1046558/37754502-8abec9f6-2ded-11e8-831e-627ceffd018e.png)

Link to a viewer (if hosted on a site other than mozilla.github.io/pdf.js or as Firefox/Chrome extension):
Default viewer from pdfjs-1.10.88-dist. 
Edit, tested on mozilla.github.io/pdf.js with the same issue

**Comment by timvandermeij** [comment]
This may be a duplicate of #7631.

**Comment by cristianmtr** [comment]
What's the status on this? I use pdf.js as part of FoundryVTT (for online RPGs)

**Comment by brendandahl** [comment]
@calixteman I vaguely remember you saying you had a fix for something like this?

**Comment by calixteman** [comment]
Yeah I've something somewhere in my stack.

**Comment by cristianmtr** [comment]
Any news on this?

**Comment by STBaf** [comment]
Any news on this ?

**Comment by marco-c** [comment]
Once #15060 lands, we should make sure the issues duplicated to this are actually all fixed.

**Comment by marco-c** [comment]
I checked all duplicates, they were all fixed!
